Ginger/GingerCoreNET/SourceControl/SourceControlIntegration.cs

```diff
@@ -192,10 +192,13 @@ public static bool ResolveConflicts(SourceControlBase SourceControl, string path
                     path = path.Replace("@/", "\\");
                 }
                 RepositoryFolderBase repositoryFolderBase = null;
-                if (path != SourceControl.SolutionFolder)
+                if (path.EndsWith("xml"))
```

Comment:
Please check if it will handle other file extension from documents folder of solution

Reply:
yes, it is handling conflict in text-based formats in Documents, like TXT, XML and Json. only issue is with non text files like .db